features/employment/mtr/employment.feature

```diff
@@ -0,0 +1,30 @@
+Feature:
+    "Employment"
+
+    Scenario: The client is employed, and receiving a benefit in kind
```

Comment:
the scenario (and test) seem to say more than I would expect as we're only showing MTR employment threshold